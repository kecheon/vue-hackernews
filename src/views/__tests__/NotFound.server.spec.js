/**
 * @jest-environment node
 */

import { renderToString, render } from '@vue/test-utils'
import NotFound from '../NotFound.vue'

test('renders correctly ', () => {
  const markupString = renderToString(NotFound)
  expect(markupString).toMatchSnapshot()
})

test('renders 404 inside <h1> tag', () => {
  const wrapper = render(NotFound)
  expect(wrapper.find('h1').text()).toBe('404')
})
