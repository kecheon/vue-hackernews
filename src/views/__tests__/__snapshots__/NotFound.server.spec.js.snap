// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders time correctly on server  1`] = `"<!---->"`;
